src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,94 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.github.lalyos.jfiglet.FigletFont;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.cli.*;
+
+import java.lang.reflect.Method;
+
+import static com.github.vkorobkov.jfixturescmd.Prompt.getProperties;
+
+@Slf4j
+class CmdArgs {
+    private String[] args = null;
+    private Options options = new Options();
+
+    CmdArgs(String[] args) {
+        this.args = args;
+        this.options.addOption("y", "srcYaml", true, "set source YAML folder path");
+        this.options.addOption("s", "outputSql", true, "set output SQL file path");
+        this.options.addOption("t", "sqlType", true, "set SQL type: " + SqlTypes.getValues().toLowerCase());
+        this.options.addOption("h", "help", false, "print help message");
+        this.options.addOption("verbose", "be extra verbose");
+        this.options.addOption("version", "print the version information and exit");
```

Comment:
Let's always print `version` on startup? With usual small font under the big 'JFixtures' caption